packages/pds-ember/addon/components/pds/logomark/consul.hbs

```diff
@@ -0,0 +1,11 @@
+<Pds::Logomark
+  ...attributes
+  data-test-logomark-consul
+  aria-label="Consul logomark"
+>
+  <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 128 128">
+    <path d="M61.644 77.206A13.375 13.375 0 1174.999 63.81a13.334 13.334 0 01-13.355 13.395zM87.67 69.99a6.179 6.179 0 116.18-6.16 6.141 6.141 0 01-6.18 6.16zM110.236 75.817a5.983 5.983 0 11.117-.489c0 .137 0 .294-.117.49zM105.875 59.92a6.175 6.175 0 01-6.447-2.385 6.18 6.18 0 1111.023-5.026 6.239 6.239 0 010 2.405 5.865 5.865 0 01-4.634 5.005M127.815 75.133a5.729 5.729 0 11.118-.567 1.723 1.723 0 00-.118.567zM122.79 59.646a6.064 6.064 0 01-6.906-6.807 6.066 6.066 0 014.8-5.128 6.061 6.061 0 017.249 6.44 6.161 6.161 0 01-5.143 5.495zM118.488 96.486a6.257 6.257 0 11.763-2.542 5.475 5.475 0 01-.763 2.542zM116.259 39.818a6.182 6.182 0 01-8.892-3.48 6.182 6.182 0 011.855-6.605 6.177 6.177 0 019.403 1.657 5.71 5.71 0 01.743 3.578 6.161 6.161 0 01-3.109 4.85z"/>
+    <path d="M61.878 125.974a61.976 61.976 0 01-28.283-6.885A61.977 61.977 0 01.668 73.064a61.987 61.987 0 0198.95-58.334l-7.567 9.777a49.551 49.551 0 100 78.745l7.567 9.992a61.427 61.427 0 01-37.74 12.73z"/>
+  </svg>
```

Comment:
I think these paths may need some classes so the logo mark can be colored properly (the part inside of the C is darker than the C).

Reply:
Per the [Consul primary logo](https://drive.google.com/file/d/1NgV7MMOOhbjF89B5ELr4Fjqp42mA_d9A/view?usp=sharing) the logomark is 100% solid of the same color.

Reply:
...also the black [logomark](https://drive.google.com/file/d/1ukZWRH4A1xzKYSi71D1WNxMwwOkN_6HA/view?usp=sharing)

Reply:
I think it's still a requirement to have the color versions - they're more prominently used than the monochrome for anything other than Vault or HashiCorp logos.